cli/pkg/providers/aws/aws.go

```diff
@@ -51,6 +52,17 @@ func (p *provider) GetOverrides() tmpl.Variables {
 	}
 }
 
+func (p *provider) IsEnvironmentValid() (bool, error) {
+	necessaryBinaries := [2]string{"aws", "terraform"}
```

Comment:
Maybe it is better to define these outside of the function in a const. That way the function becomes re-usable.

Reply:
I see. So we do not make it provider specific. I was thinking if ever need to check specific things such as environment variables etc. this could be helpful. But then this function would be focused on only checking binaries (which is highly reusable)